moler/device/junipergeneric.py

```diff
@@ -34,12 +34,15 @@ def __init__(self, sm_params, name=None, io_connection=None, io_type=None, varia
         :param variant: External-IO connection variant
         :param io_constructor_kwargs: additional parameters for constructor of selected io_type
         :param initial_state: Initial state for device
+        :param lazy_cmds_events: set False to load all commands and events when device is initialized, set True to load
+                        commands and events when the first time the are required.
```

Comment:
when the first time the are required --> when they are required for the first time

Reply:
Done